jsonparser.h

```diff
@@ -0,0 +1,18 @@
+#ifndef JSONPARSER_H
+#define JSONPARSER_H
+
+#include <map>
+#include <string>
+#include <istream>
+
+class JSONParser
+{
+private:
+    static bool isNumeric(const std::string &input);
+    static std::string getData(const std::string &line);
+public:
+    static std::map<std::string, std::string> Parse(std::istream &iStream);
+    static std::map<std::string, std::string> Parse(const std::string &iString);
```

Comment:
Tudunk tehát folyamból olvasni, és meg lehet adni egy fájl nevét is, még ha a függvény neve nem is nagyon utal erre. Viszont az egyetlen stringben lévő JSON tartalmat még nem tudjuk elemezni.